friends/src/components/AddNewFriend.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+const AddNewFriend =(props)=> {
+    
+    return (
+        <form>
+            <input
+            type="text"
+            placeholder="...name"
+            name="newFriendName"
+            onChange={props.onChange}
+            />
+
+            <input
+            type="text"
+            placeholder="...age"
+            name="newFriendAge"
+            onChange={props.onChange}
+            />
```

Comment:
Don't forget to set the `value` attribute for this input (perhaps something like `value={props.newFriendAge}`).